generators/page/templates/vue/src/test/javascript/spec/app/pages/page.component.spec.ts.ejs

```diff
@@ -0,0 +1,73 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+/* tslint:disable max-line-length */
+import { shallowMount, createLocalVue, Wrapper } from '@vue/test-utils';
+import sinon, { SinonStubbedInstance } from 'sinon';
+
+import * as config from '@/shared/config/config';
+import <%= pageName %>Component from '@/pages/<%= pageFolderName %>/<%= pageFolderName %>.vue';
+import <%= pageName %>Class from '@/pages/<%= pageFolderName %>/<%= pageFolderName %>.component';
+import <%= pageName %>Service from '@/pages/<%= pageFolderName %>/<%= pageFolderName %>.service';
+
+const localVue = createLocalVue();
+
+config.initVueApp(localVue);
+<%_ if (enableTranslation) { _%>
+const i18n = config.initI18N(localVue);
+<%_ } _%>
+const store = config.initVueXStore(localVue);
+localVue.component('font-awesome-icon', {});
+localVue.component('b-alert', {});
+localVue.component('b-badge', {});
+localVue.directive('b-modal', {});
+localVue.component('b-button', {});
+localVue.component('router-link', {});
+
+describe('Component Tests', () => {
+  describe('<%= pageName %> Component', () => {
+    let wrapper: Wrapper<<%= pageName %>Class>;
+    let comp: <%= pageName %>Class;
+    let <%= pageInstance %>ServiceStub: SinonStubbedInstance<<%= pageName %>Service>;
+
+    beforeEach(() => {
+      <%= pageInstance %>ServiceStub = sinon.createStubInstance<<%= pageName %>Service>(<%= pageName %>Service);
+
+      wrapper = shallowMount<<%= pageName %>Class>(<%= pageName %>Component, {
+        store,
+        <%_ if (enableTranslation) { _%>
+        i18n,
+        <%_ } _%>
+        localVue,
+        provide: {
+          <%= pageInstance %>Service: () => <%= pageInstance %>ServiceStub
+        }
+      });
+      comp = wrapper.vm;
+    });
+
+    it('should be a Vue instance', () => {
+      expect(wrapper.isVueInstance()).toBeTruthy();
+    });
+
+    it('Should call "method" on init', async () => {
+      // THEN
+      expect(<%= pageInstance %>ServiceStub.method.callCount).toEqual(1);
```

Comment:
Could you replace it by:
 `expect(<%= pageInstance %>ServiceStub. method.called).toBeTruthy();`?